x-pack/plugins/fleet/common/constants/plugin.ts

```diff
@@ -7,3 +7,4 @@
 
 export const PLUGIN_ID = 'fleet' as const;
 export const INTEGRATIONS_PLUGIN_ID = 'integrations' as const;
+export const SECURITY_SOLUTION_ID = 'securitySolution' as const;
```

Comment:
Could we use [this](https://github.com/elastic/kibana/blob/main/packages/core/application/core-application-common/src/default_app_categories.ts#L13) instead of creating a new const?

Reply:
thanks for pointing that out 👍